include/sharg/validators.hpp

```diff
@@ -479,37 +477,17 @@ class file_validator_base
  *
  * \remark For a complete overview, take a look at \ref argument_parser
  */
-template <typename file_t = void>
```

Comment:
In the documentation above (I cannot point to it as there is no change) there is still a paragraph about the seqan3 io file that has to be removed. also check the other documentations.